ruby/test/integration/minitest_redis_test.rb

```diff
@@ -329,6 +329,68 @@ def test_junit_reporter
       END
     end
 
+    def test_redis_reporter_failure_file
+      failure_file = File.expand_path('../../fixtures/tmp/failure_file.json', __FILE__)
```

Comment:
Based on the build errors the `tmp` folder doesn't seem to exist. Have you considered using [Tempfile](https://ruby-doc.org/stdlib-2.4.0/libdoc/tempfile/rdoc/Tempfile.html) or [mktmpdir](https://ruby-doc.org/stdlib-2.5.3/libdoc/tmpdir/rdoc/Dir.html#method-c-mktmpdir)?